lib/game.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Game do
+  @win_combos [
+    {1, 2, 3}, 
+    {4, 5, 6},
+    {7, 8, 9},
+    {1, 4, 7},
+    {2, 5, 8},
+    {3, 6, 9},
+    {1, 5, 9},
+    {3, 5, 7}
+  ]
+
+  def current_player(board) do
+    if Integer.mod(Board.turn_count(board), 2) == 0, do: "X", else: "O"
```

Comment:
It'd be nice to extract these `"X"` and `"O"` token strings into constants that could be reused.